src/main/java/seedu/agendum/logic/parser/Parser.java

```diff
@@ -39,13 +39,22 @@
 
     private static final Pattern SCHEDULE_ARGS_FORMAT = Pattern.compile("(?:.+?(?=(?:(?:by|from|to)\\s|$)))+?");
 
+    private static final Pattern ALIAS_ARGS_FORMAT = Pattern.compile(
+            "(?<commandword>[\\p{Alnum}\\p{Punct}]+)\\s+(?<shorthand>[\\p{Alnum}\\p{Punct}]+)");
```

Comment:
i think symbols might break the code in someway, if they do think of some really devious methods